### PR TITLE
PE-547: Add dev tools to node-simple-server-example (put script and readme instructions regarding how to bundle)

### PR DESCRIPTION
**Pull request by jdmedlin1**
## 📝 Description
The node simple server example should support tools that promote ease of development. This includes the put script to move files to a specified player and the instructions to do so.

## 📋 List of Changes

- Please provide a concise list of the main changes.

## 🧪 Steps to Test

1. Step 1

## ✔️ Dev Complete Checklist

- [ ] PR template filled out
- [ ] Change is tested by submitter
- [ ] PR follows all linting and coding standards
- [ ] Github Issue exists (if applicable)
- [ ] Team member has been assigned
- [ ] At least one commit message is in [Conventional Commit](https://www.conventionalcommits.org/) format

### CODE REVIEW

**Comment by mingjun7** [comment]
I think the put script is generic to upload files to the player correct? If so, can it be put outside of any directories, but can be reused across the repo?

File: examples/node-simple-server-example/README.md [hunk 1]
```diff
@@ -22,11 +22,41 @@ npm run build
...
+export PLAYER_PW="<password, which is the serial number by default>"
+# e.g. export PLAYER_PW="abcd"
+```
+
+To actually put code on your player:
+```sh
+npm run cp
+```
+
+If you are confident the code will compile with webpack, you can combine building the app and deploying the app with:
```
Comment (blshukla) [inline examples/node-simple-server-example/README.md:50]: If this is not true, then what are the steps to build and deploy separately?

**Review by blshukla — COMMENTED** [review]
Can you update the description for all sections? Remove the ones that are not relevant.

**Comment by jdmedlin1** [comment]
> I think the put script is generic to upload files to the player correct? If so, can it be put outside of any example directories, and can be reused across the repo?

If we want to split the templates and/or examples into their own repos, we will want each project to have its own script to push to player. Not all the scripts are identical as well given the nature of different application scaffolding structures.

**Comment by mingjun7** [comment]
> > I think the put script is generic to upload files to the player correct? If so, can it be put outside of any example directories, and can be reused across the repo?
> 
> If we want to split the templates and/or examples into their own repos, we will want each project to have its own script to push to player. Not all the scripts are identical as well given the nature of different application scaffolding structures.

I think the best way might be we point them to use brightsign-action to push file. Therefore user do not need to load a script for all projects.

**Comment by jdmedlin1** [comment]
Can I get this merged in the meantime? I created a ticket to replace current instructions with CLI instructions.

**Review by mingjun7 — APPROVED** [review]

**Comment by bhastings1019** [comment]
> I think the best way might be we point them to use brightsign-action to push file. Therefore user do not need to load a script for all projects.

Agreed - leveraging `bsc` as an option can be added.

**Comment by jdmedlin1** [comment]
> > I think the best way might be we point them to use brightsign-action to push file. Therefore user do not need to load a script for all projects.
> 
> Agreed - leveraging `bsc` as an option can be added.

We can make this an effort for next sprint: https://brightsign.atlassian.net/browse/PE-561

**Pull request merged**